libraries/standard/http/cbmc/proofs/HTTPClient_Send/Makefile

```diff
@@ -11,25 +29,27 @@ INCLUDES +=
 # function pointer targets, leading to faster execution time.
 REMOVE_FUNCTION_BODY += __CPROVER_file_local_http_client_c_findHeaderFieldParserCallback
 REMOVE_FUNCTION_BODY += __CPROVER_file_local_http_client_c_findHeaderValueParserCallback
+# The memory safety of the http-parser callbacks are proven separately. 
```

Comment:
This is future work.

Reply:
Might also add a comment that we need to confirm the assumptions make in these future proofs are satisfied by the calling contexts in this code.  (For the combination to be sound.)

Reply:
Sure . I will add that to SIM